src/controllers/midi/midicontroller.cpp

```diff
@@ -199,6 +200,16 @@ void MidiController::commitTemporaryInputMappings() {
 
 void MidiController::receive(unsigned char status, unsigned char control,
                              unsigned char value, mixxx::Duration timestamp) {
+    if (userActivityInhibitTimer.isNull()) {
```

Comment:
I think we can just start the timer in the Contructorand remove this block here. (The same in Controller)